webapp/web/configuration.css

```
@@ -23,24 +23,28 @@
     padding: 0px;
 }
 
-.configure-package__tag-item {
+.tag-list__tag-item {
     list-style-type: none;
     border-color: #ddd;
     border-bottom-style: solid;
     border-width: 1px;
     padding: 10px;
 }
 
-.configure-package__tag-item-active {
+.tag-list__tag-item_active {
```

I think this should be:
```suggestion
.tag-list__tag-item--active {
```
(`active` seems to be a modifier on the tag item)

---

Corrected.